### PR TITLE
Bootstrap styles

### PR DESCRIPTION
**Pull request by RomaSerenkov**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,4 +26,10 @@
...
 
 /.php_cs
-/.php_cs.cache
\ No newline at end of file
+/.php_cs.cache
+###> symfony/webpack-encore-bundle ###
+/node_modules/
+/public/build/
+npm-debug.log
+yarn-error.log
```
Comment (artsems) [inline .gitignore]: You don't need any `.log` lines here since you have added `*.log`.

Comment (artsems) [inline .gitignore]: You don't need any `.log` lines here since you have added `*.log`.

File: assets/styles/global.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@import "~bootstrap/scss/bootstrap";
```
Comment (artsems) [inline assets/styles/global.scss]: Newline.

Comment (artsems) [inline assets/styles/global.scss]: Newline.

**Review by artsems — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (RomaSerenkov) [reply .gitignore]: Resolved

Comment (RomaSerenkov) [reply .gitignore]: Resolved

File: assets/styles/global.scss (hunk 2 above)
Comment (RomaSerenkov) [reply assets/styles/global.scss]: Resolved

Comment (RomaSerenkov) [reply assets/styles/global.scss]: Resolved

**Review by artsems — APPROVED** [review]

**Pull request merged**